lib/fastlane/actions/update_url_schemes.rb

```diff
@@ -0,0 +1,68 @@
+require 'plist'
+
+module Fastlane
+  module Actions
+    class UpdateUrlSchemesAction < Action
+      def self.run(params)
+        path = params[:path]
+        url_schemes = params[:url_schemes]
+
+        hash = Plist.parse_xml(path)
+        hash['CFBundleURLTypes'].first['CFBundleURLSchemes'] = url_schemes
```

Comment:
On a side note: I would love actions to be able to take a trailing block. Imagine this general purpose plist action:

```ruby
update_plist 'path/to/Info.plist' do |plist_hash|
  # Do what you want to with the hash here, and return the new hash
  # the action then replaces the file with a plist version of your new hash
  updated_plist_hash
end
```

Now, it could be built with `lambda` or `proc` arguments but it would look a little unreadable and I haven't seen any such actions so far.